### DISCUSSION TITLE
Improve flow in create/take offer process

### DISCUSSION THREAD

**Discussion by HenrikJannsen** [bisq-network/bisq#6824] [General]
When creating or taking an offer the user gets the choice to fund the trade from an external wallet for potentially better privacy (avoid coin merge if inputs from ext. wallet are kept separated). From feedback it seems that most users are not using that option and then they need to do that extra click on the "Fund from internal Bisq wallet" button. 
We should offer a way to avoid that extra step if the user chooses to use the internal wallet. We could show at the first time a popup with background info why funding from an external wallet could improve privacy and if the user prefers to not use that to give them an option to skip that step in future. In the settings we could add a toggle to re-enable it again.

Beside that there might be further improvements which could help to reduce the number of user inputs in the process. E.g. the review screen might get skipped if user wants to.

**Comment by HenrikJannsen** [comment]
@jmacxx Would you be available to work on that?

**Comment by ghost** [comment]
- Step 1 is data entry/initiate.
- Step 2 is fund the offer/trade with BTC.
- Step 3 is review before committing to the offer/trade.
- Step 4 is final review before committing to the offer/trade.

For users who choose a streamlined flow, perhaps steps 2 and 3 can be skipped .

Even step 4 could be skipped, making it a one-screen flow.


---


<details>
<summary>Make offer step 1 - 4 screenshots</summary>

![offer_1](https://github.com/bisq-network/bisq/assets/47253594/c1185b6b-c253-4f02-9480-1e4b2fb4de2c)

![offer_2](https://github.com/bisq-network/bisq/assets/47253594/e21c0a23-7b7f-46c7-b6cf-d8b5d8d50904)

![offer_3](https://github.com/bisq-network/bisq/assets/47253594/4516ef3b-2b76-492b-b5e6-b50e246524e5)

![offer_4](https://github.com/bisq-network/bisq/assets/47253594/5885436c-58d3-46c4-8e95-3773e39ac1df)

</details>


<details>
<summary>Take offer steps 1-4 screenshots</summary>



![takeoffer_1](https://github.com/bisq-network/bisq/assets/47253594/752bf757-7876-4921-a5be-9d1a52888f53)

![takeoffer_2](https://github.com/bisq-network/bisq/assets/47253594/bad8079b-dbf7-48eb-9b4b-f0acd933b178)

![takeoffer_3](https://github.com/bisq-network/bisq/assets/47253594/3d854259-55f1-4ad6-8310-ebf0458df8c3)

![takeoffer_4](https://github.com/bisq-network/bisq/assets/47253594/267e1114-0697-4d23-8280-d8345c9d067a)





</details>
